src/main/java/net/snowflake/client/core/IncidentUtil.java

```diff
@@ -71,6 +68,8 @@ public static String oneLiner(String prefix, Throwable thrown) {
   public static void dumpVmMetrics(String incidentId) {
     PrintWriter writer = null;
     try {
+      File dmpFileDirectory = new File(EventUtil.getDumpPathPrefix());
+      dmpFileDirectory.mkdirs();
```

Comment:
Do we need to worry about cleaning this directory or it's users responsibility?

Reply:
No, this is the user's responsibility for cleaning along with the dmp files produced. The user can call this function but it's never called anywhere in the codebase currently.